test/controllers/autocompleters_controller_test.rb

```diff
@@ -59,7 +59,7 @@ def test_autocomplete_location
     expect.unshift({ name: "M", id: 0 })
     expect.sort_by! { |loc| [loc[:name], -loc[:id]] }
     expect.uniq! { |loc| loc[:name] }
-    good_autocompleter_request(type: :location, string: "Modesto")
+    good_autocompleter_request(type: :location, string: "M")
```

Comment:
By changing the test to the first letter, you may be missing something resulting from the change in `classes/autocomplete.rb` above, which is now taking the whole string rather than the first letter.